CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff

```diff
@@ -384,7 +384,7 @@
       </trans-unit>
       <trans-unit id="Credits" xml:space="preserve">
         <source>Credits</source>
-        <target state="translated">Credits</target>
+        <target state="translated">Credits-Szene</target>
```

Comment:
This better fits the meaning of the header here. Does it fit width-wise, though?

Reply:
Tune in to find out!

(I'll issue a TestFlight in a little bit)